frontend/src/app/pages/settings-page/settings-page.component.html

```diff
@@ -16,18 +16,22 @@
                     </mat-expansion-panel-header>
 
                     <mat-form-field>
-                        <input matInput placeholder="First name" type="text" [formControl]="userName" required readonly>
-                        <mat-error *ngIf="userName.invalid">{{getUserNameErrorMessage()}}</mat-error>
+                        <input matInput placeholder="First name" type="text" [formControl]="userFirstName" required>
+                        <mat-error *ngIf="userFirstName.invalid">{{getUserNameErrorMessage()}}</mat-error>
```

Comment:
After your previous changes these are now inconsistent. Function name should be getUserFirstNameErrorMessage()